src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+
+  &__month {
+    display: flex;
+    gap: $gap-between-days;
+    flex-wrap: wrap;
+    max-width: calendarWidth(
+      $day-size,
+      $gap-between-days,
+      $calendar-side-paddings
+    );
+    padding: 20px $calendar-side-paddings;
+
+    @for $index from 28 through 30 {
+      &--month-length--#{$index} :nth-child(n+#{$index + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    position: relative;
+    @include squere(100px);
+    border: 1px solid #000;
+    background-color: #eee;
+    transition: all 0.5s;
```

Comment:
don't use it like transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md
![image](https://user-images.githubusercontent.com/91411692/222660044-dda468a4-85a2-47a4-8abc-3445598601b8.png)

```suggestion
    transition: transform $transitionDuration;
```